### PR TITLE
 Added a parallax feature , Added the about and services section component for the Jabbar.

### PR DESCRIPTION
**Pull request by Aditya30ag**
…rallex feature

## Important Checklist:
- [ ] You have prettyfied your code (`npx prettier --write .`)
- [ ] Linted the code using ESLint (`npx eslint . --ext .js,.jsx,.ts,.tsx`)
- [ ] This code is best you could write in terms of quality (we don't mind even if its bad :P)
- [ ] You have attached the screenshot of visible chnages in GUI, if any
- [x] You have no merge conflicts, you have pulled and rebased your PR branch
- [ ] All CI tests have passed
- [ ] **The PR is made for `develop` branch and not main branch**
- [ ] Explicitly ask for review upon submitting a chnage, so that the reviewer is notified of changes (ask @g4ze for review by default)

### CODE REVIEW

**Comment by g4ze** [comment]
kindly follow the pr template :)

**Comment by Aditya30ag** [comment]
Ok I will follow the pr template from now.

**Comment by Aditya30ag** [comment]
I think there is no issue on merging?

**Comment by Aditya30ag** [comment]
Screenshot 2024-10-27 093558

**Comment by Aditya30ag** [comment]
Screenshot 2024-10-27 093613

**Comment by g4ze** [comment]
To follow the PR template, you'll have to check the points that you've complied to.
As per the PR, in the media, kindly show the changes, via recording in your case as it is not visible by just screen shots. I hope you understand what I'm trying to say. Ty

**Comment by g4ze** [comment]
Please be a bit more descriptive about your encorporated changes

**Comment by Aditya30ag** [comment]
Screenshot 2024-10-27 233559
so in the above screen shot there are 3 errors , which was resolve by making this changes
image
I am using the lastest version of next js so in that it was showing error so i change it, was i need to go with the odd version or the lastest version of next.js

**Comment by g4ze** [comment]
You will have to use the version specified originally, as a lot of things, including deployment platform, is set accordingly. Updating might break the deployment.

**Comment by Aditya30ag** [comment]
In the "new" commit. I have add a about page and set URLs in Navbar for about, services, contact,

https://github.com/user-attachments/assets/3990c9c6-091a-4b7c-a433-1f1993c89a76

this is How about page look like

**Comment by g4ze** [comment]
Home, about, services and contact will be on the same page, activating themselves lazily with scroll. we are not making different pages for it. We are making different pages for: team, all events.

**Comment by Aditya30ag** [comment]
https://github.com/user-attachments/assets/763832dc-5d36-40ac-9b72-fd6fde86b407

I have added now correctly as you say you can see in the video

**Comment by g4ze** [comment]
Following the PR template, kindly make this or for the develop branch.

File: src/app/page.jsx [hunk 1]
```diff
@@ -64,6 +106,16 @@ export default function Home() {
         <EventsSection events={eventsData} />
       </div>
 
+      {/* About Section */}
+      <div id="aboutsection" className="flex flex-col items-center justify-center min-h-screen p-4 pb-10 gap-8 sm:p-100 text-center">
+        <About/>
+      </div>
+
+      {/* About Section */}
```
Comment (g4ze) [inline src/app/page.jsx:114]: Bad comment

File: src/components/Loader.jsx [hunk 2]
```diff
@@ -10,8 +10,8 @@ export default function Loader() {
           <Image
             src={public_path+"/assets/logo_gdg.png"}
             alt="GDG Logo"
-            width={40}
-            height={20}
```
Comment (g4ze) [inline src/components/Loader.jsx:14]: Not a required change

File: src/components/Navbar.jsx [hunk 3]
```diff
@@ -85,36 +93,35 @@ export default function Navbar() {
             <ul className="flex flex-col p-4 md:p-0 mt-4 font-medium md:space-x-8 rtl:space-x-reverse md:flex-row md:mt-0 md:border-0 ">
               <li>
                 <a
-                  href="#"
-                  className="block py-2 px-3 text-white  rounded md:bg-transparent md:text-blue-700 md:p-0 md:dark:text-blue-500"
+                  href="/"
```
Comment (g4ze) [inline src/components/Navbar.jsx:96]: Is using href even required?

**Review by g4ze — CHANGES_REQUESTED** [review]

**Comment by g4ze** [comment]
Also in the PR desc, kindly mention that you've also added section components for the Jabbar, as it is not mentioned anywhere.
Some changes in the review then we're gtg.

File: src/components/Loader.jsx (hunk 2 above)
Comment (Aditya30ag) [reply src/components/Loader.jsx]: ok i will redo the changes

File: src/app/page.jsx (hunk 1 above)
Comment (Aditya30ag) [reply src/app/page.jsx]: Sorry ,i have corrected it now

Comment (Aditya30ag) [reply src/app/page.jsx]: in the lastest commit

Comment (Aditya30ag) [reply src/app/page.jsx]: Screenshot 2024-11-01 092754
corrected it now

File: src/components/Loader.jsx (hunk 2 above)
Comment (Aditya30ag) [reply src/components/Loader.jsx]: corrected it

**Comment by Aditya30ag** [comment]
Now i have done the changes that you had mention to correct it

**Comment by g4ze** [comment]
I have reverted this merge. I need you to change the target branch of this pr to develop and not main

**Comment by g4ze** [comment]
image
change this` main` to `develop`

**Comment by Aditya30ag** [comment]
image
i Have can to develop branch

**Pull request merged**